`CHAINERX_ENABLE_LAPACK=0` causes error

In chainer root, `CHAINER_BUILD_CHAINERX=1 CHAINERX_ENABLE_LAPACK=0 MAKEFLAGS=-j8 pip install .` causes compilation error.
`CHAINER_BUILD_CHAINERX=1 CHAINERX_ENABLE_LAPACK=1 MAKEFLAGS=-j8 pip install .` doesn't.
I investigated where `CHAINERX_ENABLE_LAPACK` is defined, but I couldn't find.
I cannot understand why `CHAINERX_ENABLE_LAPACK=1` succeeds compilation.
```    [ 94%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/scalar.o
    /tmp/pip-req-build-f5leulh5/chainerx_cc/chainerx/python/routines.cc: In lambda function:
    /tmp/pip-req-build-f5leulh5/chainerx_cc/chainerx/python/routines.cc:362:63: error: ‘CHAINERX_ENABLE_LAPACK’ was not declared in this scope
      362 |     mlinalg.def("_is_lapack_available", []() -> bool { return CHAINERX_ENABLE_LAPACK; });
          |                                                               ^~~~~~~~~~~~~~~~~~~~~~
    [ 96%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/shape.o
    [ 96%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/slice.o
    [ 98%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/strides.o
    [ 98%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/testing/device_buffer.o
    [ 98%] Building CXX object chainerx/python/CMakeFiles/_core.so.dir/testing/testing_module.o
    make[3]: *** [chainerx/python/CMakeFiles/_core.so.dir/build.make:245: chainerx/python/CMakeFiles/_core.so.dir/routines.o] Error 1
    make[3]: *** Waiting for unfinished jobs....
    make[2]: *** [CMakeFiles/Makefile2:2668: chainerx/python/CMakeFiles/_core.so.dir/all] Error 2
    make[1]: *** [CMakeFiles/Makefile2:2680: chainerx/python/CMakeFiles/_core.so.dir/rule] Error 2
    make: *** [Makefile:775: _core.so] Error 2
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-f5leulh5/setup.py", line 198, in <module>
        setup(**setup_kwargs)
      File "/home/okada/PFN/chainer_check_lapack/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.7/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/lib/python3.7/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/home/okada/PFN/chainer_check_lapack/venv/lib/python3.7/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/usr/lib/python3.7/distutils/command/install.py", line 545, in run
        self.run_command('build')
      File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/usr/lib/python3.7/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/tmp/pip-req-build-f5leulh5/chainerx_build_helper.py", line 44, in run
        self.build_extension(ext)
      File "/tmp/pip-req-build-f5leulh5/chainerx_build_helper.py", line 95, in build_extension
        ['cmake', '--build', '.'] + build_args, cwd=self.build_temp)
      File "/usr/lib/python3.7/subprocess.py", line 347, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release', '--', '_core.so']' returned non-zero exit status 2.
    
    ----------------------------------------
Command "/home/okada/PFN/chainer_check_lapack/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-req-build-f5leulh5/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-0b3omalh/install-record.txt --single-version-externally-managed --compile --install-headers /home/okada/PFN/chainer_check_lapack/venv/include/site/python3.7/chainer" failed with error code 1 in /tmp/pip-req-build-f5leulh5/
You are using pip version 19.0.3, however version 19.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.```

--- a/chainerx_cc/CMakeLists.txt
+++ b/chainerx_cc/CMakeLists.txt
@@ -224,7 +224,6 @@ if(${CHAINERX_ENABLE_LAPACK})
         add_definitions(-DCHAINERX_ENABLE_LAPACK=1)
     else()
         message(STATUS "LAPACK not found")
-        add_definitions(-DCHAINERX_ENABLE_LAPACK=0)
     endif()
 endif()
 

--- a/chainerx_cc/chainerx/python/routines.cc
+++ b/chainerx_cc/chainerx/python/routines.cc
@@ -359,7 +359,11 @@ void InitChainerxLinalg(pybind11::module& m) {
     m.def("dot", [](const ArrayBodyPtr& a, const ArrayBodyPtr& b) { return MoveArrayBody(Dot(Array{a}, Array{b})); }, "a"_a, "b"_a);
 
     pybind11::module mlinalg = m.def_submodule("linalg");
-    mlinalg.def("_is_lapack_available", []() -> bool { return CHAINERX_ENABLE_LAPACK; });
+#if CHAINERX_ENABLE_LAPACK
+    mlinalg.def("_is_lapack_available", []() -> bool { return true; });
+#else
+    mlinalg.def("_is_lapack_available", []() -> bool { return false; });
+#endif
     mlinalg.def(
             "solve", [](const ArrayBodyPtr& a, const ArrayBodyPtr& b) { return MoveArrayBody(Solve(Array{a}, Array{b})); }, "a"_a, "b"_a);
     mlinalg.def("inv", [](const ArrayBodyPtr& a) { return MoveArrayBody(Inverse(Array{a})); }, "a"_a);

